azext_iot/central/_help.py

```diff
@@ -390,6 +390,18 @@ def _load_central_monitors_help():
             az iot central app monitor-properties --app-id {app_id} -d {device_id}
     """
 
+    helps[
+        "iot central app validate-properties"
+    ] = """
+        type: command
+        short-summary: Validate reported properties sent to IoT Central app.
```

Comment:
Validate in what way? How is this command supposed to work/what is the user expecting? Our commands should be as self describing as possible.